packages/conf-llvm/conf-llvm.15/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+maintainer: "Kate <kit.ty.kate@disroot.org>"
+authors: "The LLVM team"
+homepage: "http://llvm.org"
+bug-reports: "https://llvm.org/bugs/"
+license: "MIT"
+build: [
+  ["bash" "configure.sh" version]
+]
+depends: [
+  "conf-bash" {build}
+]
+depexts: [
+  ["llvm@15"] {os-distribution = "homebrew" & os = "macos"}
+  ["llvm-15"] {os-distribution = "macports" & os = "macos"}
+  ["llvm-15-dev" "zlib1g-dev"] {os-family = "debian"}
```

Comment:
```suggestion
  ["llvm-15-dev" "zlib1g-dev" "libzstd-dev"] {os-family = "debian"}
```
Debian bug once again, this time on Debian 13 and newer only (stable works fine for some reason)

cc @sylvestre